src/python/turicreate/toolkits/object_detector/_tf_image_augmenter.py

```diff
@@ -41,39 +33,36 @@
   'min_eject_coverage': 0.5
 }
 
-def hue_augmenter(image, annotation,
+def hue_augmenter(image, annotation, tf_seed,
                   max_hue_adjust=_DEFAULT_AUG_PARAMS["max_hue_adjust"]):
 
-    # Sample a random rotation around the color wheel.
```

Comment:
I don't know what `hue_adjust` is used there so I delete it haha.

Reply:
you'd better ask the team.